### ISSUE TITLE
The azure-logs wodle does not redirects the errors to ossec.log

### ISSUE THREAD

**Issue by dariommr** [wazuh/wazuh#12985]
|Wazuh version|Component|
|---|---|
| 4.3.0-40308 | Azure Wodle |

## Rationale
The azure-logs wodle is not redirecting the Error logs to the `ossec.log` even in debug mode.

## Current Results
In the configuration I set this:
```
        <request>
            <tag>audit-logs</tag>
            <query>AuditLogs | where OperationVersion contains "1"</query>
            <workspace>XXXXXXXXXXXX</workspace>
            <time_offset>24h</time_offset>
        </request>
```

This shows this error running the script manually in verbose mode:
```
04/01/2022 12:18:39 PM ERROR: AZURE Log Analytics: 400 Client Error: Bad Request for url: https://api.loganalytics.io/v1/workspaces/bc.....cbe/query?query=AuditLogs+%7C+where+OperationVersion+contains+1+%7C+order+by+TimeGenerated+asc+%7C+where+TimeGenerated+%3E%3D+datetime%282022-03-24T04%3A18%3A37.723445Z%29+
```

But if I search for this error in `ossec.log`:
```
2022/04/01 12:07:08 wazuh-modulesd:azure-logs[175092] wm_azure.c:54 at wm_azure_main(): INFO: Module started.
2022/04/01 12:07:08 wazuh-modulesd:azure-logs[175092] wm_azure.c:74 at wm_azure_main(): INFO: Starting fetching of logs.
2022/04/01 12:07:08 wazuh-modulesd:azure-logs[175092] wm_azure.c:81 at wm_azure_main(): INFO: Starting Log Analytics collection for the domain 'wazuh.com'.
2022/04/01 12:07:08 wazuh-modulesd:azure-logs[175092] wm_azure.c:120 at wm_azure_log_analytics(): DEBUG: Creating argument list.
2022/04/01 12:07:08 wazuh-modulesd:azure-logs[175092] wm_azure.c:164 at wm_azure_log_analytics(): DEBUG: Launching command: wodles/azure/azure-logs --log_analytics --la_id 2XXXXXXX --la_key XXXXXXXX --la_tenant_domain wazuh.com --la_tag azure-activity --la_query "AzureActivity | where Level != "Error"" --workspace XXXXXXXX --la_time_offset 1000h
2022/04/01 12:07:12 wazuh-modulesd:azure-logs[175092] wm_azure.c:181 at wm_azure_log_analytics(): INFO: Finished Log Analytics collection for request 'azure-activity'.
2022/04/01 12:07:12 wazuh-modulesd:azure-logs[175092] wm_azure.c:120 at wm_azure_log_analytics(): DEBUG: Creating argument list.
2022/04/01 12:07:12 wazuh-modulesd:azure-logs[175092] wm_azure.c:164 at wm_azure_log_analytics(): DEBUG: Launching command: wodles/azure/azure-logs --log_analytics --la_id XXXXXXXX --la_key XXXXXXXXX --la_tenant_domain wazuh.com --la_tag audit-logs --la_query "AuditLogs | where OperationVersion contains "1"" --workspace XXXXXXXXX --la_time_offset 24h
2022/04/01 12:07:14 wazuh-modulesd:azure-logs[175092] wm_azure.c:181 at wm_azure_log_analytics(): INFO: Finished Log Analytics collection for request 'audit-logs'.
2022/04/01 12:07:14 wazuh-modulesd:azure-logs[175092] wm_azure.c:83 at wm_azure_main(): INFO: Finished Log Analytics collection for the domain 'wazuh.com'.
2022/04/01 12:07:14 wazuh-modulesd:azure-logs[175092] wm_azure.c:100 at wm_azure_main(): DEBUG: Fetching logs finished.
2022/04/01 12:07:14 wazuh-modulesd:azure-logs[175092] wm_azure.c:70 at wm_azure_main(): DEBUG: Sleeping until: 2022/04/15 02:00:00
```
No error logs are shown. The script finishes smoothly.

### Expected results
It is needed in this case that some error should be shown in the `ossec.log`, in normal mode saying something like "Error fetching logs for tag: XXXX" and in debug mode showing the details of the error.

Thank you in advance!

**Comment by CarlosRS9** [comment]
This happens because the output of the Azure module is not shown in the `ossec.log`. It has used its own logging file in `/var/ossec/logs/azure-logs.log` since its conception. If we take a look at this log file we can see that it does indeed contain the expected error message:

```
04/05/2022 09:30:24 AM INFO: AZURE Getting the data from /var/ossec/wodles/azure/last_dates.json.
04/05/2022 09:30:24 AM INFO: AZURE Azure Log Analytics starting.
04/05/2022 09:30:24 AM INFO: AZURE Log Analytics: Getting authentication token.
04/05/2022 09:30:24 AM INFO: AZURE 142defdbc912bd359b7a2cd40766baab was not found in /var/ossec/wodles/azure/last_dates.json for log_analytics
04/05/2022 09:30:24 AM INFO: AZURE Log Analytics: The search starts for query: 'AuditLogs | where OperationVersion contains 1 | order by TimeGenerated asc | where TimeGenerated >= datetime(2022-04-04T09:30:24.707666Z) '
04/05/2022 09:30:24 AM INFO: AZURE Log Analytics: Sending a request to the Log Analytics API.
04/05/2022 09:30:26 AM ERROR: AZURE Log Analytics: 400 Client Error: Bad Request for url: https://api.loganalytics.io/v1/workspaces/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/query?query=AuditLogs+%7C+where+OperationVersion+contains+1+%7C+order+by+TimeGenerated+asc+%7C+where+TimeGenerated+%3E%3D+datetime%282022-04-04T09%3A30%3A24.707666Z%29+
04/05/2022 09:30:26 AM INFO: AZURE Azure Log Analytics ending.
```


The `/var/ossec/log/azure-logs.log` file has already been deprecated in favour of using the `ossec.log` in #11938. This change is already applied for `4.4`. In addition to this, several improvements to the way Azure logging works have been implemented.

For the `4.3` it was decided to maintain the way logging works as we had for `4.2`.

**Comment by davidjiglesias** [comment]
Already fixed in master, known issue for v4.3.0
